its/plugin/pom.xml

```diff
@@ -11,7 +11,7 @@
 
   <artifactId>it-xml-plugin</artifactId>
 
-  <name>SonarXML :: IT Plugin</name>
+  <name>SonarSource XML analyzer :: IT Plugin</name>
```

Comment:
I think we should capitalize the first letter of `Analyzer` (applies everywhere)